<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_SendOn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/upload_server.ico</normaloff>:/icons/upload_server.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>27</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>порт:</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox_Port">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Время, мс:</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox_Time">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>14</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>IP:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_IP_Send">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_Quant">
           <property name="minimumSize">
            <size>
             <width>111</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>111</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Байт передано: </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Add">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/edit_add.ico</normaloff>icons/edit_add.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Delete">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/edit_remove.ico</normaloff>icons/edit_remove.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tbl_Data">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_Message">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
